test/StringProcessing/Sema/regex_literal_type_inference.swift

```diff
@@ -10,41 +10,39 @@ takesRegex('//') // okay
 let r1 = '/.(.)/'
 // Note: We test its type with a separate statement so that we know the type
 // checker inferred the regex's type independently without contextual types.
-let _: Regex<Tuple2<Substring, Substring>>.Type = type(of: r1)
+let _: Regex<(Substring, Substring)>.Type = type(of: r1)
```

Comment:
For someone to pick up these test changes as fixes instead of failures, would they need to run update-checkout, or are the tests versioned by tag?

Reply:
They would need to update-checkout and re-run build-script.